usaspending_api/references/management/commands/loadprogramactivity.py

```diff
@@ -1,30 +1,68 @@
 import logging
+import boto
+import os
+import csv
 
+from django.conf import settings
 from django.core.management.base import BaseCommand
 from django.db import transaction
 
 from usaspending_api.etl.csv_data_reader import CsvDataReader
 from usaspending_api.references.models import RefProgramActivity
 
+BUCKET_NAME = 'gtas-sf133'
+FILE_NAME = 'program_activity.csv'
+
 
 class Command(BaseCommand):
     help = "Loads program activity codes."
     logger = logging.getLogger('console')
 
     def add_arguments(self, parser):
-        parser.add_argument('file', nargs=1, help='the file to load')
+        parser.add_argument('file', nargs='?', help='the file to load')
 
     def handle(self, *args, **options):
 
         # Create the csv reader
-        csv_file = options['file'][0]
+        csv_file = options['file']
+        if not csv_file:
```

Comment:
Here we should support both local and non-local runs

Reply:
The change here supports both. The reason for the removed `[0]` is regarding line 22, the `?` only has one value or none whereas the nargs provides a list.